inc/Calculator.h

```diff
@@ -11,6 +11,12 @@
 #define CALCULATOR_H
 
 #include <iostream>
+#include <string>
 // Add your function declarations here.
 
+void printCalculations(int a, int b);
```

Comment:
Every functions declared in a header must have this. They get generated by the doxygen extension. An example is:
/**
         * @brief Construct a new Application Access Point Handler object
         *
         * @param [in] service application access point adapter handler object pointer
         * @param [out] application unique pointer to application access point object
         */